finetune/adapter.py

```diff
@@ -204,15 +208,15 @@ def train(
 
 @torch.no_grad()
 def validate(
-    fabric: L.Fabric, model: torch.nn.Module, val_data: np.ndarray, tokenizer: Tokenizer, max_seq_length: int
+    fabric: L.Fabric, model: Parrot, val_data: np.ndarray, tokenizer: Tokenizer, max_seq_length: int
 ) -> torch.Tensor:
     fabric.print("Validating ...")
     model.eval()
     losses = torch.zeros(eval_iters)
     for k in range(eval_iters):
         input_ids, targets = get_batch(fabric, val_data, max_seq_length)
         logits = model(input_ids)
-        loss = loss_fn(logits, targets)
+        loss = chunked_cross_entropy(logits, targets, chunk_size=0)
```

Comment:
Since the motivation for chunking is to reduce `backward` memory usage, we don't need it during evaluation